apps/molgenis-components/src/docs/forms/InputSearch.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+  <demo-item id="input-search" label="Search Input (extends BaseInput)">
```

Comment:
why do we have a demo-item here? Is this some magic you use elsewhere?

Reply:
commit unclear , please elaborate